website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Getting started with the dbt Semantic Layer"
+description: "Heard about dbt Metrics or the dbt Semantic Layer and curious to give them a try? Callum McCann digs into what they are, walks through an example, and discusses how they all fit together!"
+slug: getting-started-with-the-dbt-semantic-layer
+
+authors: [callum_mccann]
+
+tags: [dbt product updates]
+hide_table_of_contents: false
+
+date: 2022-07-27
+is_featured: true
+---
+
+# Getting started with the dbt Semantic Layer
+
+> TLDR: The Semantic Layer is made up of a combination of open-source and SaaS offerings and is going to change how your team defines and consumes metrics. For information on where its headed, [see Drew's post here](https://www.getdbt.com/blog/dbt-semantic-layer).
+
+In recent months, we’ve seen a lot of excitement in the community around what the dbt Semantic Layer is. To those who aren’t following this saga with the intensity of [someone watching their investments on the crypto market](https://mobile.twitter.com/scannergr1/status/1536198701215109122/photo/1), we're rolling out this new resource to help you better understand the dbt Semantic Layer and provide clarification on the following things:
```

Comment:
I think we can go one level deeper on the opening. A classic bit of JLantz wisdom - never share that you / the community is excited - _why_ are you excited. A couple ways I think we can open this (these are styles rather than exact wordings to use but might be helpful):

- Personal anecdote - Since joining dbt Labs as a Developer Experience Advocate focused on metrics I've spent a lot of time understanding what exactly the semantic layer is and how analytics engineers can interact with it. Now, I figured it was time to help get more folks up and running.

- [call back to Coalesce last year. I _think_ this is my preferred one but not strongly held] At last year's Coalesce, Drew showed us the vision for what metrics would look like in dbt. In the time since then, we've been getting the infrastructure in place to make this a reality. We wanted to share what we have today and how this fits into the bigger picture.
- [Use case focused] You know the story - your CEO in toolX and your CFO is in tool Y. They both look at revenue and see different numbers. Hate to see it. But wait - the dbt Semantic Layer can help fix this by XYZ

Reply:
Updated!